samples/snippets/job_test.py

```diff
@@ -95,7 +99,7 @@ def test_bucket():
     bucket = storage_client.create_bucket(output_bucket_name)
 
     yield bucket
-    bucket.delete(force=True)
+    # bucket.delete(force=True)
```

Comment:
Should this be commented out? If so, please add a comment with instructions for when users should uncomment it

Reply:
Good catch! Bucket should be deleted. This was from testing.